src/js/routes/about/About.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import AboutContent from '../../../static/about.html';
+
+const About = () => (
+	<div>
+		<div dangerouslySetInnerHTML={{ __html: AboutContent }} />
```

Comment:
I dig the changes, I'll update accordingly.

This code structure was to make it easy to update the HTML _outside_ of React and then commit back via source control. I can see know that was just abstraction habits of mine... 

The `dangerouslySet...` wouldn't be too dangerous if only we have access to edit the pre-compiled HTML, which is why I included it :-).